pub_sub/java/http/checkout/src/main/java/com/service/CheckoutServiceApplication.java

```diff
@@ -0,0 +1,69 @@
+package com.service;
+
+import org.json.JSONObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.core.env.Environment;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.time.Duration;
+import java.util.Random;
+import java.util.concurrent.TimeUnit;
+
+@SpringBootApplication
+public class CheckoutServiceApplication implements CommandLineRunner {
+	private static final Logger logger = LoggerFactory.getLogger(CheckoutServiceApplication.class);
+	private static final HttpClient httpClient = HttpClient.newBuilder()
+			.version(HttpClient.Version.HTTP_2)
+			.connectTimeout(Duration.ofSeconds(10))
+			.build();
+
+	private static final String PUBSUB_NAME = "order_pub_sub";
+	private static final String TOPIC = "orders";
+	private static String dapr_host;
```

Comment:
Can we have these variables as DAPR_HOST and DAPR_HTTP_PORT just to keep it consistent across.